src/main/resources/liquibase/workbench_changelog/12_5_0.xml

```diff
@@ -0,0 +1,713 @@
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="1">
+				select count(1)
+				from information_schema.routines
+				where routine_schema = database() and specific_name = 'assignCropsToSuperAdmin';
+			</sqlCheck>
+		</preConditions>
+		<comment>Drop obsolete migration procedure assignCropsToSuperAdmin replace with changeset 12.5.0-2</comment>
+		<sql dbms="mysql" splitStatements="true" >
+			drop procedure assignCropsToSuperAdmin;
+		</sql>
+	</changeSet>
+
+	<changeSet author="vanina" id="v12.5.0-1">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="users_roles" columnName="crop_name"/>
+			</not>
+		</preConditions>
+		<comment>
+			Alter table users_roles to add crop_name
+		</comment>
+		<sql dbms="mysql" splitStatements="true">
+			ALTER TABLE users_roles
+			ADD COLUMN crop_name VARCHAR(32) NULL;
+			ALTER TABLE users_roles
+			ADD INDEX fk_crop_name_idx (crop_name ASC);
+			ALTER TABLE users_roles
+			ADD CONSTRAINT fk_crop_name
+			  FOREIGN KEY (crop_name)
+			  REFERENCES workbench_crop (crop_name)
+			  ON DELETE NO ACTION
+			  ON UPDATE NO ACTION;
+		</sql>
+	</changeSet>
+
+	<changeSet author="vanina" id="v12.5.0-2">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="users_roles" columnName="workbench_project_id"/>
+			</not>
+		</preConditions>
+		<comment>
+			Alter table users_roles to add workbench_project_id
+		</comment>
+		<sql dbms="mysql" splitStatements="true">
+			ALTER TABLE users_roles
+			ADD COLUMN workbench_project_id INT(10) UNSIGNED NULL DEFAULT NULL;
+			ALTER TABLE users_roles
+			ADD CONSTRAINT fk_workbenck_project
+			FOREIGN KEY (workbench_project_id)
+			REFERENCES workbench_project (project_id)
+			ON DELETE NO ACTION
+			ON UPDATE NO ACTION;
+		</sql>
+	</changeSet>
+
+	<changeSet author="vanina" id="v12.5.0-3">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="users_roles" columnName="created_by"/>
+			</not>
+		</preConditions>
+		<comment>
+			Alter table users_roles to add created_by
+		</comment>
+		<sql dbms="mysql" splitStatements="true">
+			ALTER TABLE users_roles
+			ADD COLUMN created_by INT(11) NULL;
+
+			ALTER TABLE users_roles
+			ADD INDEX fk_users_roles_2_idx (created_by ASC);
+			ALTER TABLE users_roles
+			ADD CONSTRAINT fk_users_roles_2
+			FOREIGN KEY (created_by)
+			REFERENCES users (userid)
+			ON DELETE NO ACTION
+			ON UPDATE NO ACTION;
+		</sql>
+	</changeSet>
+
+	<changeSet author="vanina" id="v12.5.0-4">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="users_roles" columnName="created_date"/>
+			</not>
+		</preConditions>
+		<comment>
+			Alter table users_roles to add created_date
+		</comment>
+		<sql dbms="mysql" splitStatements="true">
+			ALTER TABLE users_roles
+			ADD COLUMN created_date TIMESTAMP NULL DEFAULT NULL;
+
+			alter table users_roles
+			  modify COLUMN created_date TIMESTAMP NULL default CURRENT_TIMESTAMP;
```

Comment:
Why are we doing a **MODIFY** column in the changeset? Can't we add the default `CURRENT_TIMESTAMP` when we add the column?